packages/flutter_tools/lib/src/isolated/web_server_utlities.dart

```diff
@@ -12,14 +12,16 @@ import '../base/logger.dart';
 import '../dart/package_map.dart';
 import '../web_template.dart';
 
+/// Logs [event] to [logger].
+///
+/// The `event.level` property determines whether the event will
+/// be logged as an Error (SEVERE), a warning (WARNING) or a trace
+/// (everything else).
 void log(Logger logger, logging.LogRecord event) {
   final String error = event.error == null ? '' : 'Error: ${event.error}';
   if (event.level >= logging.Level.SEVERE) {
     logger.printError('${event.loggerName}: ${event.message}$error', stackTrace: event.stackTrace);
   } else if (event.level == logging.Level.WARNING) {
-    // Temporary fix for https://github.com/flutter/flutter/issues/109792
-    // TODO(annagrin): Remove the condition after the bogus warning is
-    // removed in dwds: https://github.com/dart-lang/webdev/issues/1722
     if (!event.message.contains('No module for')) {
```

Comment:
This `if (!event.message.contains(...)) {...}` can be removed as well, but we'll still want the `logger.printWarning(...)` statement. We just want to log warning messages unconditionally.